### ISSUE TITLE
EPUB Exports: ARIA role errors in toc and copyright pages

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4445]
The TOC should use a `<nav>` element (not a `<div>`) with both `epub:type="toc"` and the corresponding ARIA `role="doc-toc"`. The current `<div id="toc">` provides no semantic landmark for assistive technolgy users.

To validate:
- add `id="toc-heading"` to the `<h1>` and a`ria-labelledby="toc-heading"` to the `<nav>` 
- add `epub:type` attributes to `<li>` or `<a>` elements

The Copyright page: the `<section>` is semantically empty.

To validate:
- add a (visually hidden?) heading
- add `epub:type="copyright-page"` to the section

**Comment by SteelWagstaff** [comment]
If we confirm the changes we want to make, this should be pretty straightforward to implement. Here's the blade template used to generate the EPUB (and PDF) table of contents: https://github.com/pressbooks/pressbooks/blob/4ef0ff44e90e41a1968948cfd121cbd5c515c2eb/templates/export/toc.blade.php#L1-L2

Here's the copyright page blade template: https://github.com/pressbooks/pressbooks/blob/dev/templates/export/copyright.blade.php

If we need to add different markup depending on the file export type, we'll need to introduce some conditional logic that doesn't currently exist.
